create_derivatives/tests.py

```diff
@@ -77,3 +78,33 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class PDFMakerTestCase(TestCase):
+
+    def setUp(self):
+        tmp_path = Path(settings.TMP_DIR)
+        if not tmp_path.exists():
+            tmp_path.mkdir(parents=True)
+
+    def set_up_bag(self, fixture_directory, bag):
```

Comment:
Shouldn't this method be part of `setUp`? And then you set `self.bag_id` there as well?